### DISCUSSION TITLE
Blender integration still planned?

### DISCUSSION THREAD

**Discussion by cosmical2021** [InteractiveComputerGraphics/SPlisHSPlasH#149] [General]
Hi, i read this pdf where future plans include blender integration:
 https://interactivecomputergraphics.github.io/SPH-Tutorial/slides/07_SPlisHSPlasH.pdf
 "Future  GPU-based pressure solver  Integration in Blender"
 Is there still any blender integration plans on road map?

**Comment by janbender** [comment]
Yes but it will take some time since we are no Blender experts and some of the particle stuff in Blender is not well documented. To do some first steps in this direction, we wrote a better particle (and geometry) importer for Blender that we will publish soon. Moreover, we are working on a Blender plugin to set up simulation scenes for SPlisHSPlasH in Blender. With those tools you could set up the simulation in Blender, start the simulation externally and then import the data in Blender. The next big step is then to also perform the simulation in Blender.
